Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -54,7 +54,14 @@ class MemoryViewTable final : public QTableWidget
     setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
     setShowGrid(false);
     setContextMenuPolicy(Qt::CustomContextMenu);
-    setSelectionMode(SingleSelection);
+    // Selection will be set progmatically. User will still get an outline on clicked items.
+    setSelectionMode(NoSelection);
+    setTextElideMode(Qt::TextElideMode::ElideNone);
+    setStyleSheet(QStringLiteral("QTableView::item:selected"
```

Comment:
Just curious, does the default stylesheet not support `selection-color` or is there a separate reason we wanted to change the colors?  This probably only looks ok on the default theme that Dolphin ships with.

Reply:
Yeah using stylesheets are a worry with themes and stuff.  Without this style sheet it becomes light gray when the table loses focus. I was trying to keep the color constant, without coloring the cell itself, because un-coloring it is a hassle.

Not sure if there's another option. I could just remove it.

Reply:
Actually, I went back and did find another way.